rules/windows/registry/registry_set/registry_set_disable_fonction_user.yml

```diff
@@ -24,7 +25,12 @@ detection:
         Details: 'DWORD (0x00000001)'
     selection_set_0:
         EventType: SetValue
-        TargetObject|endswith: 'SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\shutdownwithoutlogon'
+        TargetObject|endswith:
+            - 'SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\shutdownwithoutlogon'
+            - 'SOFTWARE\Microsoft\Windows\CurrentVersion\PushNotifications\ToastEnabled'
+            - '\SYSTEM\CurrentControlSet\Control\Storage'
```

Comment:
endswith with '\SYSTEM\CurrentControlSet\Control\Storage'  alone should never math .

Reply:
Yep, nice catch was a typo on my part from copying the line below :) Thanks.